apis/facebook/api/src/objects/fb_user.e

```diff
@@ -0,0 +1,223 @@
+note
+	description: "Object representing a Facebook User."
+	date: "$Date$"
+	revision: "$Revision$"
+	EIS: "name=Facebook User", "src=https://developers.facebook.com/docs/graph-api/reference/user/", "protocol=uri"
+
+class
+	FB_USER
+
+feature -- Access: Basic Properties
+
+	id:  detachable STRING
+			-- The id of this person's user account.
+
+	name: detachable STRING
+			-- The person's full name.
+			--! Predeterminded.
+
+	birthday: detachable STRING
```

Comment:
if this is a string ... what is the value ?  ISO 8601  ? unix timestamp ? else?

Maybe it could be DATE_TIME ?